src/containers/App/App.js

```diff
@@ -4,23 +4,25 @@ import './App.css';
 import { connect } from 'react-redux';
 import  {getChuck}  from '../../actions/mocks.actions';
 import DatePicker from 'antd/lib/date-picker';  // for js
-import 'antd/lib/date-picker/style/css';        // for css
+import 'antd/lib/date-picker/style/css';
```

Comment:
This needs to be imported in App.css so it get applied over all the project. Already did on my branch